public/engine/ivmodelinfo.h

```diff
@@ -65,8 +65,10 @@ class IVModelInfo
 
 	// Returns name of model
 	virtual const char				*GetModelName( const model_t *model ) const = 0;
-	virtual vcollide_t				*GetVCollide( const model_t *model ) const = 0;
+	virtual vcollide_t				*GetVCollideScaled( int modelindex, float flScale ) const = 0;
+	virtual vcollide_t				*GetVCollideScaled( const model_t *model, float flScale ) const = 0;
 	virtual vcollide_t				*GetVCollide( int modelindex ) const = 0;
+	virtual vcollide_t				*GetVCollide( const model_t *model ) const = 0;
 	virtual void					GetModelBounds( const model_t *model, Vector& mins, Vector& maxs ) const = 0;

```

Comment:
Are you positive that the two existing GetVCollide functions got reordered? In the Windows binaries, overloads with the same name will always be in reverse order in the binary compared to the header.

Reply:
When looking at CModelInfoClient I see that the biggest part of the class moved by 2 functions (https://i.imgur.com/NpKWzHA.png). As I reversed the CMdlCache first I knew they added new functions to GetVCollide and at CModelInfoClient it seemed to fit. After deeper research in these functions for example at GetVCollide at index 4 I see that it has many similarities to the one from se2007 (https://i.imgur.com/uwRIai8.png) The same applies for the next function (GetVCollide at index 5, https://i.imgur.com/FXfafUi.png) GetVCollide at index 6 & 7 are just wrapper to 4 & 5, but they have 1 less parameter and they call the function at index 4/5 with 1.0f as parameter which is the model scale (https://i.imgur.com/IyUuIzB.png).

Reply:
Oh, after reading your comment again now I understand what you mean. I will check it and fix my pull request afterwards if its the case.